homeassistant/components/hive/alarm_control_panel.py

```diff
@@ -49,6 +50,7 @@ class HiveAlarmControlPanelEntity(HiveEntity, AlarmControlPanelEntity):
     _attr_supported_features = (
         AlarmControlPanelEntityFeature.ARM_NIGHT
         | AlarmControlPanelEntityFeature.ARM_AWAY
+        | AlarmControlPanelEntityFeature.TRIGGER
```

Comment:
Do all hive alarms support this, or does it need to be conditioned, for example depending on the model of the alarm?

Reply:
@emontnemery hive only sell one alarm model so yes all alarms will have this feature